newscoop/src/Newscoop/NewscoopBundle/Form/Type/PublicationOutputSettingsType.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * @package Newscoop\NewscoopBundle
+ * @author Paweł Mikołajczuk <pawel.mikolajczuk@sourcefabric.org>
+ * @copyright 2014 Sourcefabric z.ú.
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt
+ */
+
+namespace Newscoop\NewscoopBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class PublicationOutputSettingsType extends AbstractType
```

Comment:
@m038 - why we need it? Looks like duplicate of existing PublicationSettingsType